Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "LineFilledGraphWidget.h"
+#include "ImPlotImplementation.h"
+
+void SFilledGraphWidget::Construct(const FArguments& InArgs)
+{
+	InData = InArgs._InDataArray;
+
+	ChildSlot
+		[
+			SAssignNew(ImGuiCustomWidget, SImGuiCustomWidget)
+			.ContextIndex(InArgs._ContextIndex)
+			.OnImGuiDrawDelegate(this, &SFilledGraphWidget::OnImGuiDraw)
+		];
+}
+
+void SFilledGraphWidget::OnImGuiDraw() const
+{
+	//ImPlot::ShowDemoWindow();
+	//ImGui::ShowDemoWindow();
+	//return;
+
+	constexpr auto WindowFlags = ImGuiWindowFlags_NoDecoration
+		| ImGuiWindowFlags_NoBackground
+		| ImGuiWindowFlags_NoSavedSettings;
+
+	constexpr auto GraphFlags = ImPlotFlags_CanvasOnly | ImPlotFlags_AntiAliased;
+
+	constexpr auto AxisFlags = ImPlotAxisFlags_NoDecorations;
+
+	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
+
+	ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
+	ImGui::SetNextWindowSize(ImVec2(AllottedGeometry.GetLocalSize().X, AllottedGeometry.GetLocalSize().Y), ImGuiCond_Always);
+	ImGui::Begin("ImPlot Demo", NULL, WindowFlags);
+
+	// ---------------------------------
+	{
+		TArray<float> XValues;
+		for (float f = 0; f < InData.Num(); ++f)
+		{
+			XValues.Add(f);
+		}
+
+		ImPlot::SetNextPlotLimits(0, 100, 0, 500, ImGuiCond_Always);
```

Comment:
What are these magic numbers 100 and 500? If they do stuff it might be worth passing them in as const or constexpr variables to make it a bit more obvious?
```suggestion
                constexpr int32 Whatever100Means = 100;
                constexpr int32 Whatever500Means = 500;
		ImPlot::SetNextPlotLimits(0, Whatever100Means , 0, Whatever500Means , ImGuiCond_Always);
```